src/Apple/Provider.php

```diff
@@ -119,11 +122,11 @@ public function userByIdentityToken(string $token): User
      */
     public static function verify($jwt)
     {
-        $jwtContainer = Configuration::forSymmetricSigner(
-            new AppleSignerNone,
-            AppleSignerInMemory::plainText('')
-        );
-        $token = $jwtContainer->parser()->parse($jwt);
```

Comment:
I know the documentation from lcobucci/jwt talks about this `Configuration` object for easier use, but the only things it needs to do for the current use case is to provide a parser and a validator.

The parser only has a simple depencency on the `JoseEncoder` class, the validator is free of other dependencies. So it does not make sense to deal with providing a signer for "none" (explained above that this is scary because it might be insecure), and a `Key` class with an empty key that are never going to be used for token validation.